### DISCUSSION TITLE
Whats is going on with hosseini.ninja?

### DISCUSSION THREAD

**Discussion by charesch** [HandyOrg/HandyControl#1120] [Q&A] [answered]
It was a great documentation but it seems offline. What is happening?

**Comment by ghost1372** [comment, marked as answer]
Hi @charesch
Unfortunately the domain (hosseini.ninja) has expired and I can not pay and renew it.
But you can access docs with original domain
Ghost1372.github.io

**Reply by charesch** [reply]
Thank you very much!
